package/utils/nand-utils/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2009-2016 OpenWrt.org
```

Comment:
Omit the copyright notice or leave it at the original value if you copied it from somewhere else.